### ISSUE TITLE
Watchface on Amazfit Bip Lite

### ISSUE THREAD

**Issue by confusionists** [Freeyourgadget/Gadgetbridge#1682]
#### Before requesting a new feature, please confirm the following:
- [ ] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [ ] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [ ] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your issue is: installing a watchface in Amazfit BIP Lite
*If applicable, please attach [logs](https://github.com/Freeyourgadget/Gadgetbridge/wiki/Log-Files)*

#### Your wearable device is: Amazfit BIP Lite, Firmware Version: 1.1.6.32

*Please specify model and firmware version if possible*

#### Your Gadgetbridge version is: 0.37.0

*New requests about already solved/documented topics could be closed without further comments. Same for too generic or incomplete reports.*

I habe successfully paired my amazfit bip lite device through server based pairing with gadgetbridge. I had to open the database file as databank file on my mobile phone. It was the only way to read out the auth key. Now I won't to install a new watch face with a *.bin file. Gadgetbridge tells me the device is not ready. Latest changes in Gadgetbridge app tell that firmware updates are not possible yet. Installing watchfaces is like installing a new firmware. Is this the reason why I could not install a new watchface? Will it be possible soon when installing firmware is possible? Is it not supported yet from gadgetbridge to install watchfaces on an Amazfit Bip? Too many questions, I know. ;-) hope someone can help. Thanks.

**Comment by ashimokawa** [comment]
Yes, next version will support that. Just wanted to prevent people flashing the wrong firmware and have some testers for the connection stuff. Thanks for testing ;)

**Comment by vanous** [comment]
@confusionists thank you very much for a clear confirmation that the server based pairing is required, much appreciated!

**Comment by ashimokawa** [comment]
@vanous
Still not sure if the initial firmware (1.1.5.xx) on the bip light is without server based pairing...

**Comment by confusionists** [comment]
Thanks for the info. I am looking forward to the the next Gadgetbridge app version which is supporting flashing firmwares and watchfaces too.

Just a small tip how I paired my Amazfit Bip with Gadgetbridge. I followed this guide:

https://techwiser.com/gadgetbridge/

But when reading out the author key I opened the "journal" file as database file on my mobile phone. I don't remember exactly the path to the auth key in database file but I think it was filed under device, then I had to scroll to to right until the Auth key was seen. I don't know the exact path anymore. Be sure to add "0x" to this key when adding to Gadgetbridge as explained in the link. This way server based pairing worked for me. Hope it helps.
